internal/npm_install/test/golden/node_modules/@gregmagolan/test-a/BUILD.bazel.golden

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
I think each generated file needs some comment at the top pointing to what generated it

Reply:
It does. The golden files strip all the comments out for comparison.